mods/default/nodes.lua

```diff
@@ -1766,188 +1766,276 @@ minetest.register_node("default:lava_flowing", {
 -- Tools / "Advanced" crafting / Non-"natural"
 --
 
-local chest_formspec =
-	"size[8,9]" ..
-	default.gui_bg ..
-	default.gui_bg_img ..
-	default.gui_slots ..
-	"list[current_name;main;0,0.3;8,4;]" ..
-	"list[current_player;main;0,4.85;8,1;]" ..
-	"list[current_player;main;0,6.08;8,3;8]" ..
-	"listring[current_name;main]" ..
-	"listring[current_player;main]" ..
-	default.get_hotbar_bg(0,4.85)
-
-local function get_locked_chest_formspec(pos)
+local function get_chest_formspec(pos)
```

Comment:
Mind if I do that in a subsequent PR? The locked formspec was never public to begin with. I have no issues against doing so, but I'd prefer to do it additionally so it can get some proper discussion.